scipy/spatial/transform/rotation.pyx

```diff
@@ -1016,6 +1019,106 @@ cdef class Rotation(object):
         else:
             return cls(quat, normalize=False, copy=False)
 
+    @classmethod
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    def from_mrp(cls, mrp):
+        """Initialize from Modified Rodrigues Parameters (MRPs).
+
+        MRPs are a 3 element attitude description commonly used in the aerospace
+        industry. It is similar to the rotation vector, except instead of scaling
```

Comment:
Maybe just following definition in `from_rotvec` is a good idea.

MRPs is a 3 dimensional vector co-directional to the axis of rotation and whose magnitude is equal to ``tan(theta / 4)``, where ``theta`` is the angle of rotation (in radians) [1].